client/state/posts/selectors.js

```diff
@@ -34,23 +34,32 @@ export function getPost( state, globalId ) {
 }
 
 /**
- * Returns a normalized post object by its global ID.
+ * Returns a normalized post object by its global ID, or null if the post does
+ * not exist. A normalized post includes common transformations to prepare the
+ * post for display.
  *
- * @param  {Object} state    Global state tree
- * @param  {String} globalId Post global ID
- * @return {Object}          Post object
+ * @param  {Object}  state    Global state tree
+ * @param  {String}  globalId Post global ID
+ * @return {?Object}          Post object
  */
 export const getNormalizedPost = createSelector(
 	( () => {
-		// Cache normalize flow in immediately-invoked closure so to avoid
+		// Cache normalize flow in immediately-invoked closure to avoid
 		// regenerating same flow on each call to this selector
 		const normalize = flow( [
```

Comment:
>this seems overly fancy. why not just store it as a module variable and avoid the IIFE entirely?

Probably will, especially if other selectors start calling this directly (related discussion at https://github.com/Automattic/wp-calypso/pull/6242#discussion_r68131443 ). At the time, seemed an easy way to keep the selector self-contained.